lib/support/changelog.js

```diff
@@ -12,6 +12,11 @@ var dateFormat = require('dateformat');
 var helper = require('./misc');
 
 var changelog = module.exports = {
+  needles: [
+    '## Upcoming',
+    '## Unreleased'
```

Comment:
None. Just a matter of personal taste (the original keepachangelog actually promotes `Unreleased`)